Bullwinkle.class.nut

```diff
@@ -242,11 +246,14 @@ class Bullwinkle {
         if (handler != null) {
 
             // Invoke the handler and delete the package when done
-            imp.wakeup(0, function() {
+            local timer = imp.wakeup(0, function() {
```

Comment:
removed the log and instead added a generic onError callback to settings that is called when too many agent timers in the mix